fastf1/api.py

```diff
@@ -787,7 +844,7 @@ def data_in_lap(lap_n):
                 drv_data['Sector2SessionTime'][i+1]:
             drv_data['Sector2SessionTime'][i+1] = new_s2_time
         if (new_s3_time := drv_data['Time'][i] + drv_data['Sector1Time'][i+1] + drv_data['Sector2Time'][i+1] +
-                drv_data['Sector3Time'][i+1]) < drv_data['Sector3SessionTime'][i+1]:
+            drv_data['Sector3Time'][i+1]) < drv_data['Sector3SessionTime'][i+1]:  # noqa: E129,E501
```

Comment:
Lint change, should not be in here. Also, I don't think noqa's should be used for a visual change like that